scripts/network-launch/clusters/ext-dev-1

```diff
@@ -0,0 +1,12 @@
+bootNodeIP="143.198.145.104"
```

Comment:
Rename ext-dev-1 -> ext-devs-1. And all other

Reply:
renamed